### ISSUE TITLE
Unable to access "Team Settings" 

### ISSUE THREAD

**Issue by Millali** [HPDCollaborative/builder23review#23]
Hi Vince, 

When I click "Team Settings" on my HPDC account, nothing happens. Please watch the video:

https://www.loom.com/share/dc38effe772e4906bac19a9479cd932b

When a login as a regular user, I can't seem to be able to manage members, the team setting link does not even show up. 

https://www.loom.com/share/725db74b0ccc4d82be019091cb10f49b

Best,

Milla

**Comment by secondmanveran** [comment]
@Millali 

Correct, not implemented because of all the additional work that was piled on.
